src/Components/Main/Pages/ChatPage/ChatForm/ChatFormContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import type { ReactElement } from 'react';
+import { memo } from 'react';
+import type { FormikHelpers } from 'formik';
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import { ChatForm } from './ChatForm';
+import type { ChannelStatus } from '../../../../../utils/types/common';
+
+const validationSchema = Yup.object().shape({
+  text: Yup.string().max(100, 'Must be not more than 100 characters').required('Required'),
+});
+
+type PropsType = {
+  sendMessage: (message: string) => void;
+  channelStatus: ChannelStatus;
+};
+type FormDataType = {
+  text: string;
+};
+
+export const ChatFormContainer = memo<PropsType>(({ sendMessage, channelStatus }): ReactElement => {
+  const onSubmit = (formData: FormDataType, { resetForm }: FormikHelpers<FormDataType>): void => {
+    const date = new Date();
+    const hours = date.getUTCHours() < 10 ? `0${date.getUTCHours()}` : `${date.getUTCHours()}`;
+    const minutes = date.getUTCMinutes() < 10 ? `0${date.getUTCMinutes()}` : `${date.getUTCMinutes()}`;
+    const time = `${hours}:${minutes} UTC+0`;
```

Comment:
Are you trying to convert the time to UTC?
If so - you may be interested in https://stackoverflow.com/questions/948532/how-to-convert-a-date-to-utc